tools/devtool

```diff
@@ -520,7 +520,7 @@ ensure_ci_artifacts() {
 
     # Fetch all the artifacts so they are local
     say "Fetching CI artifacts from S3"
-    S3_URL=s3://spec.ccfc.min/firecracker-ci/v1.5/$(uname -m)
```

Comment:
It passed yesterday, lets hope it still does :pray:

Reply:
and by yesterday I mean Friday

Reply:
OK c7g / host=4.14 still needs the guest 5.10 legacy artifact. We can try the no-sve version if it works, otherwise we can copy over the v1.5 version of it.

Reply:
All tests are skipped in https://buildkite.com/firecracker/firecracker-pr/builds/6599#018b14f8-41fb-40aa-a795-210362c942a5/55-116

Reply:
Fixed up the artifacts, seems like the c7g tests are no longer skipped now (copied over the `legacy` folder from the 1.5 artifacts)